(FIXED)Lesson 2 arrays and sorting

HolnievVitalii opened the pull request.

Commit:
Lesson 1. General information about algorithms and data structures

Commit:
- an array of 10,000 elements was created
- the array is filled with random numbers
- methods are written that implement the considered types of sorting, and the speed of each is checked.

HolnievVitalii commented:
results:

Bubble sort runtime = 170 millis
Sort by selection runtime = 15 millis
Insertion sort runtime = 0 millis

den-codeman commented on Arrays and Sorting/src/Main.java:
```diff
@@ -0,0 +1,113 @@
+
+    public static void main(String[] args) {
+
+        fillArray();//  filling the array with random elements
```
После сортировки пузырьком вы сортируете уже отсортированный массив выбором и вставкой.

Commit:
Lesson 2. Arrays and sorting
- fixed error sorting an already sorted array;
- each sorting and its execution time are performed simultaneously in different threads, which does not interfere with each other.
